### PR TITLE
Bug 1247951 - In tools UI, hide secret content behind a click

### PR DESCRIPTION
**Pull request by andreadelrio**
Secret only shows after clicking a button.

### CODE REVIEW

**Comment by andreadelrio** [comment]
@djmitche What do you think of this?

File: secrets/secreteditor.jsx [hunk 1]
```diff
@@ -58,7 +59,7 @@ var SecretEditor = React.createClass({
         },
         editing:          true,
         working:          false,
-        error:            null
+        error:            null,
```
Comment (djmitche) [inline secrets/secreteditor.jsx]: Any reason to omit showSecret here?

Comment (andreadelrio) [inline secrets/secreteditor.jsx]: Yes, there's no CurrentSecretId so no secret to show.

**Comment by djmitche** [comment]
I like it!

File: secrets/secreteditor.jsx (hunk 1 above)
Comment (djmitche) [inline secrets/secreteditor.jsx]: Ah, thanks

**Pull request merged**